docs/guide/data_types/timeseries.rst

```diff
@@ -30,58 +32,51 @@ To create the sample `sunpy.timeseries.sources.goes.XRSTimeSeries`, type the fol
     >>> import sunpy.data.sample  # doctest: +REMOTE_DATA
     >>> my_timeseries = ts.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES)  # doctest: +REMOTE_DATA
 
-.. doctest-skip-all
-
 This is calling the `~sunpy.timeseries.TimeSeries` factory to create a time series from a GOES XRS FITS file.
-Note that if you have not downloaded the data already you should get an error and some instruction on how to download the sample data.
 
-The variable ``my_timeseries`` is a :ref:`timeseries` object.
+The variable ``my_timeseries`` is a `~sunpy.timeseries.GenericTimeSeries` object.
 To create one from a local GOES/XRS FITS file try the following:
 
 .. code-block:: python
 
     >>> my_timeseries = ts.TimeSeries('/mydirectory/myts.fits', source='XRS')   # doctest: +SKIP
 
 **sunpy** will attempt to detect automatically the instrument source for most FITS files.
-However timeseries data are stored in a variety of file types (FITS, txt, csv, CDF) and formats, and so it is not always possible to detect the source.
-For this reason, it is good practice to explicitly state the source for the file.
-SunPy ships with a number of known instrumental sources.
+However timeseries data are stored in a variety of file types (FITS, txt, csv, CDF), and so it is not always possible to detect the source.
```

Comment:
The usage of "timeseries" vs "time series" is inconsistent. A quick google shows that both are acceptable. I do not have an opinion one way or another, but we should be consistent.

Reply:
It looks like we mostly use "time series" so maybe it is just easier to stick with that.

Reply:
👍  thanks for catching this! I went with "time series" and fixed them all.